src/hooks/useViewState.ts

```diff
@@ -118,14 +121,24 @@ export function useViewState({
           break;
 
         // Request to close the modal, to the closed view state (which depends on whether customer wants to display the collapse view or hide it)
-        case 'close':
+        case 'close': {
           setState({
             current: dismissedViewState,
             previous: state.current,
             auth,
             firstSelectedViewState: null,
           });
+          if (savedActiveElement !== null) {
+            savedActiveElement.focus();
+          } else {
+            const tempInteractiveEl = document.createElement('span');
+            tempInteractiveEl.setAttribute('tabindex', '1');
+            document.body.prepend(tempInteractiveEl);
+            tempInteractiveEl.focus();
+            tempInteractiveEl.remove();
```

Comment:
nit: maybe a note to describe why this is needed?

also, i'm sorta confused on the last two steps? the temp element is being focused on, so that brings focus to the new temp element for the first interactive/tab indexed element on the page? why then remove? is it so the focus can then be shifted around by the user?

Reply:
ooooh thanks for explaining the removal!